huaweicloud/services/acceptance/cbr/resource_huaweicloud_cbr_vault_test.go

```diff
@@ -172,7 +174,7 @@ resource "huaweicloud_cbr_vault" "test" {
 `, basicConfig, name)
 }
 
-func testAccVault_backupServer_step2(basicConfig, name string) string {
+func testAccVault_backupServer_step2(basicConfig, name, epsId string) string {
```

Comment:
```suggestion
func testAccVault_backupServer_step2(basicConfig, name string) string {
```
there is unnecessary to append a new argument, and we can use the env in the script as follows:
```
enterprise_project_id = %[3]s
...

`, basicConfig, name, acceptance.HW_ENTERPRISE_PROJECT_ID_TEST)
```

Reply:
good idea, this suggestion is very nice